git_machete/cli.py

```diff
@@ -790,13 +790,13 @@ def strip_remote_name(remote_branch: RemoteBranchShortName) -> LocalBranchShortN
 
     except (argparse.ArgumentError, argparse.ArgumentTypeError) as e:
         print(get_short_general_usage())
-        sys.stderr.write(f"\n{e}\n")
+        print(f"\n{e}")
```

Comment:
Buuut.... shouldn't `sys.stderr` be specified as the `file = ...`? 🤔

Reply:
Coz in the current shape, the output will go to `sys.stdout`!

Reply:
Like this? `print("Error in atexit._run_exitfuncs:", file=sys.stderr)`

Reply:
Yes, let's give it a shot